### ISSUE TITLE
Implement --global flag for enable/disable commands

### ISSUE THREAD

**Issue by jcrd** [gdraheim/docker-systemctl-replacement#70]
This is the excerpt about `--global` from the `systemctl` man page:
```
--global
When used with enable and disable, operate on the global user configuration directory, thus enabling or disabling a unit file globally for all future logins of all users.
```
[Link](https://www.freedesktop.org/software/systemd/man/systemctl.html#--global)

**Comment by gdraheim** [comment]
# _o.add_option("--global", action="store_true", dest="globally", default=_globally,
     #    help="Enable/disable unit files globally") # for all user logins

The option could be activated but then it would simply be ignored.

**Comment by jcrd** [comment]
I suppose it's better to explicitly fail with this option rather than silently do nothing.

**Comment by gdraheim** [comment]
I have checked what that --global does and it seems to only have an effect for --user mode services.

The features is implemented in v1.4.3326 including a testcase for it.

Please give it a try.

**Comment by gdraheim** [comment]
no response.
